sympy/parsing/c/c_parser.py

```diff
@@ -735,6 +727,68 @@ def transform_decl_stmt(self, node):
 
             return statement
 
+        def transform_paren_expr(self, node):
+            """To handle outermost parenthesis of a well formed rhs"""
```

Comment:
```suggestion
            """Transformation function for Paranthesis expressions
            
            Returns the result from its children nodes
            
            """
```

Reply:
I think something like this might be better. 
This should handle any parentheses in an expression, not only the outermost parenthesis.

Reply:
As of now, every parenthesized expression is not handled by paren_expr since we are using shunting yard in binary operator. But of course, this docstring is better in the long term as we will implement the same using clang directly once that patch you mentioned is merged!